<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>533</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MSCV6 Software Engineering, Sandeep/Peixi</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(80, 80, 80);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Vertex Count</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="vertexCount">
          <property name="font">
           <font>
            <family>MS Reference Sans Serif</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="digitCount">
           <number>10</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
</string>
          </property>
          <property name="text">
           <string>Face Count</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="faceCount">
          <property name="font">
           <font>
            <family>MS Reference Sans Serif</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="digitCount">
           <number>10</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Light 1</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Light 2</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_3">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Light 3</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777214</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Spectral</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="1">
               <widget class="QLabel" name="label_3">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Pick an EigenVector</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QSpinBox" name="eigenSpinner">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(48, 48, 48);</string>
                </property>
                <property name="wrapping">
                 <bool>false</bool>
                </property>
                <property name="frame">
                 <bool>true</bool>
                </property>
                <property name="showGroupSeparator" stdset="0">
                 <bool>false</bool>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>1000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QFormLayout" name="formLayout_5">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QPushButton" name="btnSmooth">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(254, 254, 126);</string>
              </property>
              <property name="text">
               <string>Create Laplace</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_3">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(254, 254, 126);</string>
              </property>
              <property name="text">
               <string>Render Eigen</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Compression</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Eigen Vector Count</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="eigenComponentSpin">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QSlider" name="eigenComponentSlider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_4">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Compress</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>WaterMark</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>170</width>
          <height>112</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="status">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item row="0" column="0">
     <widget class="myGL" name="glWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave_Laplace_Matrix"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuLights">
    <property name="title">
     <string>Lights</string>
    </property>
    <widget class="QMenu" name="menuShading_Model">
     <property name="title">
      <string>Shading Model</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionFLAT"/>
     <addaction name="actionSMOOTH"/>
    </widget>
    <addaction name="actionPosition"/>
    <addaction name="menuShading_Model"/>
   </widget>
   <widget class="QMenu" name="menuMaterials">
    <property name="title">
     <string>Materials</string>
    </property>
    <addaction name="actionEdit_Material"/>
   </widget>
   <widget class="QMenu" name="menuExperimenta">
    <property name="title">
     <string>Experimental</string>
    </property>
    <addaction name="actionCurvature_Analysis"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuLights"/>
   <addaction name="menuMaterials"/>
   <addaction name="menuExperimenta"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionPosition">
   <property name="text">
    <string>Position</string>
   </property>
   <property name="toolTip">
    <string>Position of glLight</string>
   </property>
  </action>
  <action name="actionEdit_Material">
   <property name="text">
    <string>Edit Material</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionCurvature_Analysis">
   <property name="text">
    <string>Curvature Analysis</string>
   </property>
  </action>
  <action name="actionFLAT">
   <property name="text">
    <string>FLAT</string>
   </property>
  </action>
  <action name="actionSMOOTH">
   <property name="text">
    <string>SMOOTH</string>
   </property>
  </action>
  <action name="actionSave_Laplace_Matrix">
   <property name="text">
    <string>Save Matrices</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myGL</class>
   <extends>QWidget</extends>
   <header>mygl.h</header>
   <container>1</container>
   <slots>
    <signal>faceCount()</signal>
    <slot>LoadObject()</slot>
    <slot>GetLightsCount()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>eigenComponentSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>eigenComponentSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>685</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>759</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eigenComponentSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>eigenComponentSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>759</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>617</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eigenSpinner</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pushButton_3</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>634</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>765</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
